lib/Dialect/FIRRTL/FIRRTLTypes.cpp

```diff
@@ -388,6 +388,59 @@ bool firrtl::areTypesEquivalent(FIRRTLType destType, FIRRTLType srcType) {
   return destType.getWidthlessType() == srcType.getWidthlessType();
 }
 
+/// Returns whether the two types are weakly equivalent.
```

Comment:
You might mention that weak type equivalence comes from section 4.6 Weak Type Equivalence

Reply:
Will update on the header definition. (Which one is supposed to get the documentation?)